dhcpcd-hooks/65-kano-dhcp-hook

```diff
@@ -35,14 +34,24 @@ REBOOT | BOUND)
 
     systemd-run /usr/bin/kano-network-hook $reason
     systemd-run /usr/bin/kano-dashboard-sysupdates "bound" "$ip_addr"
+
+    # Try uploading the tracking data to our servers
+    # Should be quiet on failure
+    kano-sync --upload-tracking-data --silent &
+
+    # Sync objects from the content API in the background
+    # TODO: This should be removed once we have the daemon done
+    sudo kano-content sync &
```

Comment:
Do we believe this might cause us any problems? Running something with `sudo` from this hook?

Reply:
Update on this: seems like kano-content is able to write files with the correct permissions so it seems like it's working as expected.

Reply:
Ouch, most probably `sudo` is not needed here, the hooks run as root already

Reply:
I've double checked and you're absolutely right - `sudo` is not required and I've removed it now.